crates/store/src/genesis/config/samples/01-simple.toml

```diff
@@ -1,6 +1,11 @@
 timestamp = 1717344256
 version   = 1
 
+
+[fee_parameters]
+symbol                = "MIDEN"
+verification_base_fee = 0
+
 [[fungible_faucet]]
 decimals     = 3
 max_supply   = 100_000_000
```

Comment:
Let's create an issue for refactoring this to be more inline with something like what is described in https://github.com/0xMiden/miden-node/pull/1143#discussion_r2265981740.